app/routes.py

```diff
@@ -0,0 +1,79 @@
+from flask import Flask
+from flask import jsonify
+from flask import request
+from flask import Response
+from flask import json
+from users_model import Users
+from blue_print import user_blueprint
+
+
+My_app = Flask(__name__, instance_relative_config=True)
+My_app.register_blueprint(user_blueprint)
+user_obj = Users()
+
+#Geta all users
+@My_app.route('/api/get-users', methods=['GET'])
+def get_all_users():
+    if user_obj.return_users():
+        return jsonify(user_obj.return_users())
+    else:
+        return jsonify ({"Empty user list" : "Please add user"})
+
+
+@My_app.route("/api/add-user", methods=['POST'])
+def add_user():
+    bol = True
+    if request.data:
+        user_details= request.json
+        if user_obj.validate_user_obj(user_details):
+            user_name = user_details['user_name']
+            user_pasword = user_details['user_pasword']
+            user_location = user_details['user_location']
+            user_age = user_details['user_age']
+            for user in user_obj.users_list:
```

Comment:
You can create a function that searches user lists to return value in order to avoid running the same loop(duplication)

Reply:
your ryt , thx